HelpMyStreetFE/HelpMyStreetFE/src/js/app.js

```diff
@@ -45,9 +45,10 @@ $(function () {
                     $(".postcode__info, #postcode_invalid").show();
                 } else {
                     if (data.volunteerCount == 0 && data.championCount == 0)
-                        $(".postcode__info, #postcode_notcovered").show();
```

Comment:
Changed this because it looks like jquery's `.show()` function sets the display type to be display:block, which overrides the css definition.